lib/mayaUsd/fileio/shading/shadingModeRegistry.cpp

```diff
@@ -119,6 +152,53 @@ UsdMayaShadingModeRegistry::_ListImporters() {
     return ret;
 }
 
+REGISTER_SHADING_MODE_RENDER_CONTEXT(
+    UsdMayaShadingModeTokens->preview,
+    "USD Preview Surface",
+    "Exports the bound shader as a USD preview surface UsdShade network.");
```

Comment:
I tried to merge two concepts (convertingTo and renderContext). Bad idea. Let's be explicit then:
REGISTER_SHADING_MODE_CONVERSION(
    UsdMayaShadingModeTokens->UsdPreviewSurface,  // <- What type of shading node to write 
    UsdMayaShadingModeTokens->universal, // <- renderContext to use
    "USD Preview Surface", // <- Nice name
    "Exports the bound..." // <- Tooltip
);
Would this move me out of "dicey terminology" territory?

Reply:
Ah, ok. Yeah, I think using a dedicated term like "conversion" makes a lot more sense than trying to use renderContext directly.

Do you still think it's necessary to associate a renderContext with the conversion mode when registering one? I would think that the selection of a particular shader ID to convert to would specify an exporter, and that that exporter would make the appropriate choice of renderContext based on which renderers would support that shader. So in the case of specifying conversion to UsdPreviewSurface, it would be the exporter for that shader ID that specifies the renderContext as universal rather than having to specify that as a property of the conversion mode. There may be some corner cases I'm not thinking of.

And smaller note, but for the tokens used here, I'd suggest using the existing ones provided by USD:
```C++
UsdImagingTokens->UsdPreviewSurface
UsdShadeTokens->universalRenderContext
```